src/js/_store/modules/categories.js

```diff
@@ -87,5 +99,27 @@ export default {
         commit('toggleLoading');
       });
     },
+    confirmDeleteCate({ commit }, { categoryId, categoryName }) {
```

Comment:
[nits]

```suggestion
    confirmDeleteCategory({ commit }, { categoryId, categoryName }) {
```

ですかね？？👀

Reply:
そうです！！！
しかし、関数名を confirmDeleteCategor にすると
文字列が長すぎてエラーになるためその関数名にしていました... :sob:
関数名を変更しました！:sweat_smile:
218e0d0701b204affaa80f331a91596d1858edbb

Reply:
どんなエラーがどこに出てました？
キャプチャ等を送ってもらえると助かります🙏
（自分の環境では特に出てなさそうに思えたので！）

Reply:
<img width="1389" alt="スクリーンショット 2023-07-21 17 20 54" src="https://github.com/gizumo-education/gizumo_wiki_lesson/assets/98654011/760546ff-5d4d-4258-8581-7608fc14d9dc">
こんな感じで１行９０文字以内にするルールがあると出てきたので関数名変えていた...のですが、
よく考えたら改行すればよかっただけなので関数名戻しました... :weary:
116b0beccd40cce360ae9b58a05730eafcace6af